Actions/MediaFastForwardAction.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.Media.Control;
+using SuchByte.MacroDeck.ActionButton;
+using SuchByte.MacroDeck.Plugins;
+
+
+// ReSharper disable once CheckNamespace
+namespace MediaControls_Plugin; // Don't change because of compatibility
+
+public class MediaFastForwardAction : PluginAction
+{
+    private GlobalSystemMediaTransportControlsSessionManager _manager;
+
+    public MediaFastForwardAction()
+    {
+        if (PluginManager.Plugins.ContainsKey("Macro Deck Media Controls"))
+        {
+            MacroDeckPlugin plugin = PluginManager.Plugins["Macro Deck Media Controls"];
+            if (plugin.GetType().IsEquivalentTo(typeof(MediaControlsPlugin)))
+            {
+                _manager = ((MediaControlsPlugin)plugin).Manager;
+            }
+        }
+    }
+    public override string Name => "Media Fast Forward";
+    public override string Description => "Fast forwards the current track on a media player.\n\r\n\rConfiguration: no";
+    public override void Trigger(string clientId, ActionButton actionButton)
+    {
+        var session = _manager.GetCurrentSession();
+        if (session == null)
+        {
+            return;
+        }
+        var test = Task.Run(async () => await session.TryFastForwardAsync()).GetAwaiter().GetResult();
```

Comment:
You can remove the variable and the .GetAwaiter().GetResult(); is also not necessary.

Reply:
Thank you for the help